ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -41,6 +41,41 @@ StackLayout {
 
     currentIndex:  chatsModel.activeChannelIndex > -1 && chatGroupsListViewCount > 0 ? 0 : 1
 
+    function populateSuggestions(){
+        chatInput.textInput.forceActiveFocus(Qt.MouseFocusReason)
+        chatInput.suggestionsList.clear()
+        const len = chatsModel.suggestionList.rowCount()
+
+        var idMap = {}
```

Comment:
I'm probably missing something, but it seems like you never use the `idMap`?

Reply:
I do inside the loops

Reply:
Ah ok, yeah I see now. I thought the idMap, was used as the model, but now it's just something to check the id, with is fine 👍